keygen/src/keygen.rs

```diff
@@ -346,6 +387,14 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                         .short("f")
                         .long("force")
                         .help("Overwrite the output file if it exists"),
+                )
+                .arg(
+                    Arg::with_name("derivation_path")
+                        .long("derivation-path")
+                        .value_name("ACCOUNT or ACCOUNT/CHANGE")
+                        .takes_value(true)
+                        .validator(is_derivation)
+                        .help("Derivation path to use: m/44'/501'/ACCOUNT'/CHANGE'; default key is device base pubkey: m/44'/501'/0'")
```

Comment:
Should `44'/501'` be implicit?  This is, after all, the *Solana* CLI?

Reply:
Per our conversation, this is good as is.  The only change we need is that the USB URL in the config file needs to either start with `44'/501'` or not include a derivation path at all.

Good:
```
usb://ledger
usb://ledger/nano-s
usb://ledger/nano-s/5YAHsQHFg6mfr4vhEPvbj1CWtXRDV9fKSV5U3y4hC1Ui
usb://ledger/nano-s/5YAHsQHFg6mfr4vhEPvbj1CWtXRDV9fKSV5U3y4hC1Ui/44'/501'/0
```

Bad:
```
usb://
usb://ledger/nano-s/5YAHsQHFg6mfr4vhEPvbj1CWtXRDV9fKSV5U3y4hC1Ui/0
```

Reply:
Done!